### PR TITLE
Move glam_etl files to the correct project folder

### PR DESCRIPTION
**Pull request by lucia-vargas-a**
Fix for [Bug 1879890](https://bugzilla.mozilla.org/show_bug.cgi?id=1879890).
It requires the project_id to be set when deploying the table manually and the DAG publish-new-tables to deploy the tables in this project.

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-2748)

### CODE REVIEW

**Review by edugfilho — APPROVED** [review]

**Comment by edugfilho** [comment]
Thank you very much for fixing this!

**Comment by edugfilho** [comment]
@scholtzan looking at the failing CI task I think bqetl is trying to generate a dag for the new tasks, but we actually need it to be replaced. How do we go about this?

**Review by whd — CHANGES_REQUESTED** [review]
Reiterating some things I mentioned in https://mozilla-hub.atlassian.net/browse/DSRE-1538: there are two datasets here: `moz-fx-data-glam-prod-fca7.glam_etl` and `moz-fx-data-shared-prod.glam_etl`. The [dataset_metadata.yaml](https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/glam_etl/dataset_metadata.yaml) file in shared-prod defines the syndicate and includes its configuration (a syndicate dataset is a dataset that contains `select *` views for all tables in the dataset it is syndicating, in this case `moz-fx-data-glam-prod-fca7.glam_etl`).

If you remove these files, you will be deleting `moz-fx-data-shared-prod.glam_etl`. From discussions in the Jira ticket, this might be desirable eventually, but because `workgroup_access` is only applied for `moz-fx-data-shared-prod`, this is currently how the GLAM application is able to access BQ data.

My recommendation would be:
1. keep `sql/moz-fx-data-shared-prod/glam_etl/dataset_metadata.yaml` where it is, as it is
1. for every `metadata.yaml` file under `sql/moz-fx-data-shared-prod/glam_etl/*/metadata.yaml`, reduce the metadata to `workgroup_access`, moving the rest of the metadata to  `sql/moz-fx-data-glam-prod-fca7/glam_etl/*/metadata.yaml`
1. delete `sql/moz-fx-data-glam-prod-fca7/glam_etl/dataset_metadata.yaml`: it is non-canonical and not acted upon by infrastructure

This will have the effect I think you are looking for without deleting datasets or breaking GLAM. If you're interested in going further and removing `moz-fx-data-shared-prod.glam_etl` entirely, you will need to update the application and encode the equivalent `workgroup_access` information somewhere else, likely https://github.com/mozilla-services/cloudops-infra/blob/master/projects/glam/tf/prod/envs/prod/resources/etl.tf.

File: sql/moz-fx-data-shared-prod/glam_etl/glam_fenix_release_aggregates/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+workgroup_access:
```
Comment (whd) [inline sql/moz-fx-data-shared-prod/glam_etl/glam_fenix_release_aggregates/metadata.yaml:1]: It's not obvious to me that `glam_fenix_release_aggregates` is being created anywhere. `glam_etl.glam_fenix_release_aggregates_v1` exists in both projects.

**Review by whd — APPROVED** [review]

File: sql/moz-fx-data-shared-prod/glam_etl/glam_fenix_release_aggregates/metadata.yaml (hunk 1 above)
Comment (edugfilho) [reply sql/moz-fx-data-shared-prod/glam_etl/glam_fenix_release_aggregates/metadata.yaml:1]: Thanks. I just spotted this. Bad copy/paste while working on https://github.com/mozilla/bigquery-etl/pull/5020 in parallel.

**Pull request merged**